src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -164,7 +166,7 @@ public AuthorList() {
      */
     public static AuthorList parse(String authors) {
         Objects.requireNonNull(authors);
-
+        authors = authors.replaceAll(" and", ",");
```

Comment:
Are you sure that his works in all cases?

What about

Lastname1, Jr1, Firstname1 and Lastname2, Jr2, Firstname2

(See https://tex.stackexchange.com/q/206760/9075)

This will produce three names, won't it?

Lastname2, Jr1 and Firstname1, Lastname2 and Jr2, Firstname2.

So, the quick fix is not a fix. Please add tests for normal handling of the "Jr" parts. Please google a bit for examples. The example given at https://tex.stackexchange.com/q/206760/9075 should be a good start.

Reply:
Oh, I had the same comment as @tobiasdiez 

So, please add our examples as test cases.